### PR TITLE
new page: usage statistics

### PR DESCRIPTION
**Pull request by bobertlo**

### CODE REVIEW

File: src/contributing/usage-statistics.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Usage Statistics
+
+If you would like to contribute usage reports, the
+[PopCorn](https://github.com/the-maldridge/popcorn) program reports installation
+statistics back to the Void project. PopCorn only reports which packages are
+installed, and the output of `xuname` for statistics on which platforms are
+being used.
```
Comment (Vaelatern) [inline src/contributing/usage-statistics.md]: `This does not report services installed or enabled, or anything other than installed packages. In this fashion, it avoids almost all personally identifiable information.`

Comment (bobertlo) [reply src/contributing/usage-statistics.md]: I brought some of that language in, but do not understand the difference between installed packages and installed services.

File: src/contributing/usage-statistics.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+If you would like to contribute usage reports, the
+[PopCorn](https://github.com/the-maldridge/popcorn) program reports installation
+statistics back to the Void project. These statistics are purely opt-in, the
+reporting programs are *not* installed by default on any void systems.
+
+*PopCorn* only reports which packages are installed, their version, and the host
+CPU architecture (the output of `xuname`.) This does not report which services
+are enabled, or any other personal information. Individual systems are tracked
+persistently by a random (client generated) UUID.
```
Comment (the-maldridge) [inline src/contributing/usage-statistics.md]: Please state that the purpose of this UUID is exclusively to permit accurate aggregation of statistics by ensuring each system is counted exactly once per sampling interval, which is 24 hours.

File: src/contributing/usage-statistics.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+First, install the `PopCorn` package:
+
+```
+# xbps-install PopCorn
+```
+
+And then, to run the service automatically, enable the sv script:
+
+```
+# ln -s /etc/sv/popcorn /var/service
```
Comment (the-maldridge) [inline src/contributing/usage-statistics.md]: Maybe make a note here that this will attempt to report stats once per day.

**Comment by bobertlo** [comment]
I totally get it. Let me know if there is anything else.

**Pull request merged**